xml_converter/src/string_helper.hpp

```diff
@@ -14,4 +14,12 @@ vector<string> split(string input, string delimiter);
 
 string normalize_type_name(string type_name);
 
+#ifndef _BASE64_H_
+#define _BASE64_H_
 
+typedef unsigned char BYTE;
```

Comment:
I don't actually like this typedef, lets just leave `char` as `char` even though it was taken from another source.

Reply:
Changed all instances of `BYTE` to `uint8_t`